src/styles/main.css

```
@@ -0,0 +1,236 @@
+}
+
+.btn-more {
+  background-color: #0da9fd;
+  max-width: 1280px;
+  height: 49px;
+  gap: 10px;
+  opacity: 1;
+  border-radius: 8px;
+  padding: 10px;
+  color: #ffffff;
+  font-size: 24px;
```

Stylelint (17.5.0)

[error] Unexpected duplicate "font-size" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)